### PR TITLE
Flexbar 2.5.0

### PR DESCRIPTION
**Pull request by dmaticzka**
* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [X] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/flexbar/2.5.0/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  sha256: "{{ sha256 }}"
+
+build:
+  number: 0
+  skip: True # [osx]
+
+requirements:
+  build:
+    - gcc    # [not osx]
+    - libgcc # [not osx]
```
Comment (bgruening) [inline recipes/flexbar/2.5.0/meta.yaml]: libgcc does not need to be a build dependency

**Comment by dmaticzka** [comment]
thanx @bgruening

**Comment by mdehollander** [comment]
Any plans to also build a package for the 3.0,x releases? An earlier attempt by @holtgrewe had some issues creating a successful build: https://github.com/bioconda/bioconda-recipes/pull/4088

**Comment by dmaticzka** [comment]
Compiling the 3.0.x series requires C++-14. @holtgrewe has started conda-forge recipes for gcc 5 and 6 here: conda-forge/staged-recipes#2089. Still in the early stages, though. Looks like a major undertaking...

**Comment by mdehollander** [comment]
Thanks for the information. I will keep an eye on threads like this one: https://github.com/bioconda/bioconda-recipes/issues/3224

**Pull request merged**